docs/main_text.md

```diff
@@ -301,6 +301,13 @@ This arises in data processed by refine.bio when RNA-seq and microarray data are
 To confirm that we have quantile normalized data correctly before returning results to the user, we evaluate the top half of expression values and confirm that a KS test produces a non-significant p-value.
 Users who seek to analyze RNA-seq and microarray data together should be aware that the low-expressing genes may not be comparable across the sets.
 
+#### Skipping quantile normalization for RNA-seq experiments
+
+When selecting RNA-seq samples for download and to aggregate by experiment, users have the option to skip quantile normalization by first selecting Advanced Options and checking the "Skip quantile normalization for RNA-seq samples" box.
+In this case, the output of tximport will be delivered in TSV files (see [our section on RNA-seq data processing with tximport](#tximport)). 
+These data can be used for differential expression analysis as "bias corrected counts without an offset" as described in the <a href = "https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html#use-with-downstream-bioconductor-dge-packages" target = "blank">_Use with downstream Bioconductor DGE packages_ section of tximport vignette</a>. 
+Note that these data will be less comparable to other datasets from refine.bio because this step has been skipped.
+
```

Comment:
Looks fine to me. 

If you prefer to keep how this is currently structured and want to retain it, we should add an FAQ. 

If not, I would restructure so that `When selecting RNA-seq samples for download and to aggregate by experiment, users have the option to skip quantile normalization by first selecting Advanced Options and checking the "Skip quantile normalization for RNA-seq samples" box.` is towards the end because, some users will get redirected here after they've found the advanced options.

Reply:
What would the FAQ question be?